src/components/table/demo/index.tsx

```diff
@@ -7,55 +7,81 @@ import { Component, h } from "@stencil/core"
 })
 export class TableDemo {
 	render() {
-		return [
-			<smoothly-table>
-				<smoothly-table-row>
-					<smoothly-table-header>Header A</smoothly-table-header>
-					<smoothly-table-header>Header B</smoothly-table-header>
-					<smoothly-table-header>Header C</smoothly-table-header>
-					<smoothly-table-header>Header D</smoothly-table-header>
-					<smoothly-table-header></smoothly-table-header>
-				</smoothly-table-row>
-				<smoothly-table-row>
-					<smoothly-table-expandable-cell>
-						normal row (exp.cell)
-						<div slot="detail">expandable cell 1 content</div>
-					</smoothly-table-expandable-cell>
-					<smoothly-table-expandable-cell>
-						expandable cell
-						<div slot="detail">expandable cell 2 content</div>
-					</smoothly-table-expandable-cell>
-					<smoothly-table-expandable-cell>
-						expandable cell
-						<div slot="detail">expandable cell 3 content</div>
-					</smoothly-table-expandable-cell>
-					<smoothly-table-expandable-cell>
-						expandable cell
-						<div slot="detail">expandable cell 4 content</div>
-					</smoothly-table-expandable-cell>
-				</smoothly-table-row>
+		return (
+			<Host>
```

Comment:
`Host` looks unnecessary here as you don't set any attributes and event listeners here?

Reply:
Changed to use an array instead.